### PR TITLE
Update notebooks with results of the track

### LINKED ISSUE #965: Improve notebooks with new logic of TSDataset
### 🚀 Feature Request

Now Transforms works with TSDataset, the core mechanic has changed and we need to reflect this changes in the notebooks.

### Proposal

1. Improve `Get Started` 
- Add transforms parameter to `transform/inverse_transform/make_future`
- Add `inverse_transform` step after forecast
2. Improve `Custom Transform and Model`
- Should cover the logic of adding reversible/irreversible transforms in new arch

### Test cases

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by alex-hse-repository**
## Before submitting (must do checklist)

- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Closing issues

closes #965

### CODE REVIEW

File: examples/custom_transform_and_model.ipynb [hunk 1]
```diff
@@ -618,7 +618,7 @@
     }
```
Comment (Mr-Geekman) [inline examples/custom_transform_and_model.ipynb:618]: <p>use -&gt; uses</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (Mr-Geekman) [inline examples/custom_transform_and_model.ipynb:618]: <p>I think we should describe what <code>ReversibleTransform</code> and <code>IrreversibleTransform</code>  is, and what is the difference between them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (alex-hse-repository) [reply examples/custom_transform_and_model.ipynb:618]: <p>Fixed</p>

Comment (alex-hse-repository) [reply examples/custom_transform_and_model.ipynb:618]: <p>Add the info about the difference</p>

**Comment by alex-hse-repository** [comment]
<p>Fixed</p>

---

 <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/#comment-1447724866'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: examples/custom_transform_and_model.ipynb [hunk 2]
```diff
@@ -202,6 +202,16 @@
     "ts.head(5)"
```
Comment (Mr-Geekman) [inline examples/custom_transform_and_model.ipynb:202]: <blockquote>The one thing we need to implement in both cases is only the method <code>get_regressors_info</code> — it should return the created regressors, created by the Transform. </blockquote><p>This phrase looks strange. May be we should rewrite it somehow. Created was used two times.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (Mr-Geekman) [inline examples/custom_transform_and_model.ipynb:202]: <p>Don't we show both implemented transforms and that results for them are the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (alex-hse-repository) [reply examples/custom_transform_and_model.ipynb:202]: Done

**Review by Mr-Geekman — APPROVED** [review]

**Pull request merged**